package.json

```diff
@@ -39,10 +40,14 @@
   },
   "dependencies": {
     "broccoli-funnel": "^3.0.3",
-    "broccoli-merge-trees": "^4.2.0"
+    "broccoli-merge-trees": "^4.2.0",
+    "ember-auto-import": "^1.6.0",
```

Comment:
seems a bit unfortunate that any non-ember user will now have to install webpack, but I don't know how to solve this any other way either. having a `ember-qunit-dom` package just to integrate things isn't really great either.

Reply:
Now that I have the try scenarios setup, I'll try a few other permutations to see if we can avoid it.

Reply:
just confirmed, and removed